src/core/mcis/provisioning.go

```diff
@@ -197,11 +197,13 @@ type TbVmReq struct {
 	SpecId           string   `json:"specId" validate:"required"`
 	ImageId          string   `json:"imageId" validate:"required"`
 	VNetId           string   `json:"vNetId" validate:"required"`
-	SubnetId         string   `json:"subnetId"`
+	SubnetId         string   `json:"subnetId" validate:"required"`
 	SecurityGroupIds []string `json:"securityGroupIds" validate:"required"`
 	SshKeyId         string   `json:"sshKeyId" validate:"required"`
 	VmUserAccount    string   `json:"vmUserAccount,omitempty"`
 	VmUserPassword   string   `json:"vmUserPassword,omitempty"`
+	RootDiskType     string   `json:"rootDiskType,omitempty"`
+	RootDiskSize     string   `json:"rootDiskSize,omitempty"`
```

Comment:
```suggestion
	RootDiskType     string   `json:"rootDiskType,omitempty" example:"default, TYPE1, ..."` // "", "default", "TYPE1", AWS: ["standard", "gp3"], Azure: ["PremiumSSD", "StandardHHD"], GCP: ["pd-standard", "pd-extreme"], ALIBABA: ["cloud_efficiency", "cloud_ssd"], TENCENT: ["CLOUD_PREMIUM", "CLOUD_SSD"]
	RootDiskSize     string   `json:"rootDiskSize,omitempty" example:"default, 30, 42, ..."` // "default", Integer (GB): ["50", ..., "1000"]
```

이런 식으로 라도 예시를 제공해줘야 할 것 같습니다. :)

https://docs.google.com/spreadsheets/d/1Y-GoZhr9DwCSJ8IhZk4tEPvbplQeQ3lUiKyOMjHO14Q/edit#gid=413527211